test/ExportVerilog/rtl-dialect.mlir

```diff
@@ -112,6 +114,7 @@ module {
   // CHECK-NEXT:   assign r34 = ~a;
   // CHECK-NEXT:   assign r35 = array[a+:3];
   // CHECK-NEXT:   assign r36 = {3{a}};
+  // CHECK-NEXT:   assign r37 = array[a];
```

Comment:
Please add another test for a two dimensional subscript, that would turn into `array2d[x][y]`.